src/inner_product_round.rs

```diff
@@ -52,7 +52,7 @@ pub struct InnerProductRound<'a, P> {
 
     // Seed for mask recovery
     round: usize,
-    seed_nonce: Option<Scalar>,
+    seed_nonce_helper: Option<Vec<u8>>,
```

Comment:
I switched from requiring the seed nonce be a `Scalar` to a more general `Vec<u8>` since there never was a cryptographic reason for the former, except to enforce some level of effective entropy.